macros/jsctypesSidebar.ejs

```diff
@@ -25,15 +25,23 @@ var text = mdn.localStringMap({
 <section class="Quick_links" id="Quick_Links">
  <ol>
   <li><a href="/<%=locale%>/docs/Mozilla/js-ctypes"><strong><em>js-ctypes</em></strong></a></li>
-
-  <li data-default-state="<%=state('Introduction')%>"><a href="/<%=locale%>/docs/Mozilla/js-ctypes/Using_js-ctypes"><%=text['Introduction']%></a>
-    <%-template("ListSubpagesForSidebar", ['/' + locale + '/docs/Mozilla/js-ctypes/Using_js-ctypes', 1])%>
+  <li class="toggle">
```

Comment:
This compiles now. On https://developer.mozilla.org/en-US/docs/Mozilla/js-ctypes/js-ctypes_reference/CType, the new sidebar (right side) has more vertical spacing than before.

![pr 789 - jsctypessidebar](https://user-images.githubusercontent.com/286017/46223782-ecd85780-c319-11e8-821d-ed83321b9af9.png). I'm OK with the change, but @schalkneethling may have something to say about it.

Reply:
Nope, all good. I like the new spacing.